attacked/functions.py

```diff
@@ -0,0 +1,36 @@
+from PIL import ImageGrab
+from io import BytesIO
+import subprocess
+import pyperclip

```

Comment:
If you have depentencies for your code.  consiuder freezing them into requirments.txt

Reply:
same for Pillow and any other one that might accoure